### PR TITLE
[DO NOT MERGE] Add one bitmap index test case.

### PR DESCRIPTION
**Pull request by yueawang**
## What changes were proposed in this pull request?

Add one bitmap index test case.

## How was this patch tested?

unit test

### CODE REVIEW

File: src/test/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapIndexSuite.scala [hunk 1]
```diff
@@ -39,9 +39,10 @@ class BitMapIndexSuite extends QueryTest with SharedSQLContext with BeforeAndAft
     sql(s"""CREATE TEMPORARY VIEW oap_test (a INT, b STRING)
             | USING oap
             | OPTIONS (path '$path')""".stripMargin)
-    sql(s"""CREATE TEMPORARY VIEW oap_test_date (a INT, b DATE)
-            | USING oap
-            | OPTIONS (path '$path')""".stripMargin)
+    val path1 = Utils.createTempDir().getAbsolutePath
+    sql(s"""CREATE TEMPORARY VIEW oap_test_1 (a INT, b STRING)
+           | USING oap
+           | OPTIONS (path '$path1')""".stripMargin)
```
Comment (chenghao-intel) [inline src/test/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapIndexSuite.scala]: Seems we are keep using the `path` in the rest of the function, why do we have to use `path1` in this particular case?

Comment (yueawang) [reply src/test/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapIndexSuite.scala]: Moved to testcase which accesses it, also renamed to a meaningful name.

**Comment by lee-lei** [comment]
Thanks Yue to submit the test case to catch the issue #406. I am analyzing the root cause. Thanks.

**Comment by linhong-intel** [comment]
Just for lei to debug, so we will not merge this.

**Comment by lee-lei** [comment]
IMHO, it's a good test case to merge in order to track either feature dev or performance optimization patch to achieve the expected functionality in the future. Thanks.

**Comment by linhong-intel** [comment]
close it and @lee-lei will upload this case in his PR

**Pull request closed without merging**